### DISCUSSION TITLE
Any way to use stability ai api for Stable Diffusion plugin?

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#2471] [Q&A]
Is there any way to use stability.ai api for the Stable Diffusion plugin?

Documentation for the API can be found here https://platform.stability.ai/docs/api-reference

**Comment by danny-avila** [comment]
Not yet, PRs welcome

**Reply by ABHIJITH-EA** [reply]
Hi, Is there any update on this feature? Is it still not possible?
